docs/content/en/docs/workflows/ci-cd.md

```diff
@@ -22,7 +22,91 @@ wait for `deployments` to stabilize and succeed only if all deployments are succ
 ## Waiting for Skaffold deployments using `healthcheck`
 {{< maturity "deploy.status_check" >}}
 
-{{< todo 1937 >}}
+`skaffold deploy` optionally performs a `healthcheck` for [`Deployment` kind](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) resources and waits for them to be stable.
+This feature is very handy to use in Continous Delivery pipeline to ensure the deployed resources are
+in healthy state before proceeding with the next steps in pipeline.
+
+{{<alert title="Note">}}
+`healthcheck` is disabled by default; it can be enabled with the `--status-check` flag.
```

Comment:
```suggestion
`healthcheck` is disabled by default; it can be enabled with the `--status-check` flag.</br>
```

Reply:
newlines are better with `</br>` - otherwise, with the extra newline, hugo renders a new paragraph